src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -66,6 +66,7 @@ public class RecordService {
   static final String CONTENT = "content";
   static final String META = "meta";
   static final String SCHEMA_ID = "schema_id";
+  static final String VERSION = "version";
```

Comment:
What about this?
```suggestion
  static final String SF_VERSION = "sf_version";
```

Reply:
Better - Made it `sf_connector_version`.